ddocs/medic-db/medic/validate_doc_update.js

```diff
@@ -37,18 +37,17 @@ function(newDoc, oldDoc, userCtx, secObj) {
   };
 
   var isDbAdmin = function(userCtx, secObj) {
-    if (hasRole(userCtx, '_admin')) {
+    if (hasRole(userCtx.roles, '_admin')) {
       return true;
     }
 
-    if (secObj.admins && secObj.admins.names &&
-        secObj.admins.names.indexOf(userCtx.name) !== -1) {
+    if (secObj.admins && secObj.admins.names && secObj.admins.names.indexOf(userCtx.name) !== -1) {
       return true;
     }
 
     if (secObj.admins && secObj.admins.roles) {
-      for (var i = 0; i < userCtx.roles; i++) {
-        if (hasRole(secObj.admins, userCtx.roles[i])) {
+      for (var i = 0; i < userCtx.roles.length; i++) {
```

Comment:
There was a bug here - without the `.length` this loop was never executing so national_admins were blocked. Found when adding unit tests for this PR.